draft-ietf-ccwg-ratelimited-increase.md

```diff
@@ -89,33 +89,41 @@ It could also occur because the receiver has limited the sender using flow contr
 In contrast, this present document concerns the increase in cwnd when a sender is rate-limited. These two topics are distinct,
-but are related, because both describe the management of the cwnd when the sender does not fully utilise the current cwnd.
-
-This document specifies a uniform rule that congestion control algorithms MUST apply and provides a recommendation that congestion control implementations ought to follow.
+but are related, because both describe the management of the cwnd when a sender does not fully utilise the current cwnd.
 
 An appendix provides an overview of the divergence in current RFCs and some current implementations regarding cwnd increase when the sender is rate-limited.
 
+# Conventions and Definitions
+
+{::boilerplate bcp14-tagged}
+
 ## Terminology
 
 This document uses the terms defined in {{Section 2 of !RFC5681}} and {{Section 3 of !RFC7661}}. Additionally, we define:
 
 - maxFS: the largest value of FlightSize since the last time that cwnd was decreased. If cwnd has never been decreased, maxFS is the maximum value of FlightSize since the start of the data transfer.
+- initcwnd: The initial value of the congestion window, also known as the "initial window" ("IW" in {{!RFC5681}}).
 
+# Increase rules {#rules}
 
-# Conventions and Definitions
+This document specifies a uniform rule that congestion control algorithms MUST apply and provides a recommendation that congestion control implementations ought to follow.
 
-{::boilerplate bcp14-tagged}
+When FlightSize < cwnd, regardless of the current state of a congestion control algorithm, the following rules apply for senders using a congestion controlled transport protocol:
 
-# Increase rules {#rules}
+1. The sender MUST initialise the maxFS parameter to initcwnd when the congestion control algorithm is started or restarted. Thereafter when the FlightSize is updated, the sender updates maxFS:
+
+~~~
+maxFS = max(FlightSize, maxFS)
+~~~
+
+The sender MUST reset the maxFS parameter to initcwnd after any adjustment that reduces the cwnd. It will then track the current FlightSize.
 
-When FlightSize < cwnd, regardless of the current state of a congestion control algorithm, senders using a congestion controlled transport protocol:
 
-1. MUST cap cwnd to be no larger than limit(maxFS).
-2. MAY restrict maxFS as min(maxFS, pipeACK), using "pipeACK" as defined in {{!RFC7661}}.
+2. The sender MUST cap cwnd to be no larger than limit(maxFS).
 
-In rule #1, the function limit() returns the maximum cwnd value the congestion control algorithm would yield by increasing for every ACK that is caused by bytes in flight, starting from the value of the maxFS parameter.
+In rule #1, the function limit() returns the maximum cwnd value the congestion control algorithm would yield by increasing for every ACK that acknowledges bytes in flight, starting from the value of the maxFS parameter.
```

Comment:
I am wondering what the clause "starting from the value of the maxFS parameter" adds?  It seems to read to me more clearly without it.

Reply:
_limit_ is a function that gets maxFS as input. A certain number of bytes are in flight when our rule is applied. Say maxFS is 20 (packets, for simplicity), and 10 are in flight, and the CC. phase is slow start. Then limit(maxFS) = 20+10 = 30. That's what I tried to convey here.

I don't know where the confusion about the need of maxFS comes from, and how to make this clearer. Thoughts?